pages/place/routers/confirm.js

```diff
@@ -35,6 +35,10 @@ module.exports = settings => form(Object.assign({
     }
     return res.redirect(req.originalUrl.replace(/\/confirm/, ''));
   },
+  editAnswers: (req, res, next) => {
+    delete req.session.form[req.model.id].validationErrors;
+    return res.redirect(req.baseUrl.replace(/\/confirm/, ''));
+  },
```

Comment:
What was this doing before? Straight-up not working?

Reply:
If you submit the confirm page without checking the declaration, then you correctly get a form error, but if you then navigate back to the previous form, the declaration error persists but the page doesn't have the content for the error snippet, so it fails to render.

Reply:
Good catch!